beacon_node/beacon_chain/tests/sync_committee_verification.rs

```diff
@@ -88,28 +89,34 @@ fn get_valid_sync_contribution(
         .head()
         .expect("should get head state")
         .beacon_block_root;
-    let sync_contributions = harness.make_sync_contributions(
+    let all_sync_contributions = harness.make_sync_contributions(
         &head_state,
         head_block_root,
         head_state.slot(),
         relative_sync_committee,
+        count,
     );
 
-    let (_, contribution_opt) = sync_contributions
-        .get(0)
-        .expect("sync contributions should exist");
-    let contribution = contribution_opt
-        .as_ref()
-        .cloned()
-        .expect("signed contribution and proof should exist");
+    // Find a subnet with at least `count` aggregates.
+    let sync_contributions = all_sync_contributions
+        .into_iter()
+        .find_map(|(_messages, contributions)| {
+            (contributions.len() == count).then(|| contributions)
```

Comment:
```suggestion
            (contributions.len() >= count).then(|| contributions)
```

This obtains the "at least" logic, however it wont obtain exactly `count`.

Reply:
I clarified the comment instead in this commit 09de7e286a5a2f01704d351f73dab558d846a616